.travis.yml

```diff
@@ -28,3 +30,6 @@ jobs:
       env: semantic-release
       before_script: npm run build
       script: npx semantic-release
+    - stage: publish-docs
```

Comment:
I didn't include an `env` key. Like found in the other stages. In fact, my editor's [Yaml linter](https://github.com/redhat-developer/vscode-yaml) didn't like the formatting of the current `env` key-values.

![yaml-lint](https://user-images.githubusercontent.com/10104630/71637899-57570200-2c05-11ea-8ad0-aeae3dfbd9dc.png)

Reply:
Using the `env` key is a hack to add a label to the Travis stages, to keep them apart easier. If there is only one step in a stage, it's not necessary